### PR TITLE
IPFS Gateway: Add `Connecting your website` section

### PR DESCRIPTION
**Pull request by thibmeu**
`Connecting your website` section was hidden. The page is useful to
describe how customers can connect their domains to Cloudflare.

### CODE REVIEW

File: products/distributed-web/src/content/ipfs-gateway/index.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ the InterPlanetary File System \(IPFS\) quickly and easily, without downloading
 any special software or giving up any storage space on your computer.
 
 Cloudflare's gateway is hosted at https://cloudflare-ipfs.com/. You can find the
-basics on IPFS and how to serve your website through our gateway there. These
+basics on IPFS and [how to serve your website](/ipfs-gateway/connecting-website) through our gateway there. These
```
Comment (Lekensteyn) [inline products/distributed-web/src/content/ipfs-gateway/index.md:12]: Shouldn't this be
```suggestion
basics on IPFS and [how to serve your website](ipfs-gateway/connecting-website) through our gateway there. These
```

for
https://developers.cloudflare.com/distributed-web/ipfs-gateway
-> 
https://developers.cloudflare.com/distributed-web/ipfs-gateway/connecting-website

(note that no trailing `/` exists)

**Review by Lekensteyn — CHANGES_REQUESTED** [review]
I think a link can break, there is also a typo in the commit message (`Connectiong` -> `Connecting`)

File: products/distributed-web/src/content/ipfs-gateway/index.md (hunk 1 above)
Comment (thibmeu) [reply products/distributed-web/src/content/ipfs-gateway/index.md:12]: I get the format from https://github.com/cloudflare/cloudflare-docs/blame/d9ad2520f59ddd7413eae2ca1b72f70e280a7962/products/distributed-web/src/content/ipfs-gateway/browsing-ipfs.md#LC11. The link there is
```markdown
[introduction](/ipfs-gateway/)
```

**Comment by thibmeu** [comment]
I updated the commit message to fix a typo

**Review by Lekensteyn — APPROVED** [review]

**Pull request merged**